spec/features/kudo_removing_after_5_minutes_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe 'Kudo spec', type: :feature do
+  let!(:kudo) { create(:kudo) }
+
+  before do
+    login_as kudo.giver, scope: :employee
+    visit root_path
+  end
+
+  it 'checks if employee can remove kudo 5 minutes after it was sent' do
```

Comment:
it would be good to check here as well, that users can delete kudo before 5 minutes time period pass

Reply:
added 🙂